### PR TITLE
BOOK-7

### PR DESCRIPTION
**Pull request by Hollerina**
- Navbar links open in the same tab

### CODE REVIEW

File: src/layouts/RootLayout.tsx [hunk 1]
```diff
@@ -15,7 +12,9 @@ if (!PUBLISHABLE_KEY) {
   throw new Error("Missing Publishable Key");
 }
 
-//BOOK-7 move the NavBar element to here to display on every page
+
+const Header = styled.header``;
```
Comment (BarneyCampbell) [inline src/layouts/RootLayout.tsx]: Is this necessary?

File: src/pages/components/NavBar.tsx [hunk 2]
```diff
@@ -1,18 +1,90 @@
...
+  background-color: green;
+  color: black;
+  padding: 20px;
+  margin-left: 10px;
+  border-bottom-color: transparent;
+  margin-bottom: -1px;
+  text-decoration: none;
+`;
+
+const ProfileContainer = styled.div``;
```
Comment (BarneyCampbell) [inline src/pages/components/NavBar.tsx:44]: Again, is this just for clear components?

File: src/layouts/RootLayout.tsx (hunk 1 above)
Comment (Hollerina) [reply src/layouts/RootLayout.tsx]: Removed

File: src/pages/components/NavBar.tsx (hunk 2 above)
Comment (Hollerina) [reply src/pages/components/NavBar.tsx:44]: yes will be styled later on

**Review by BarneyCampbell — APPROVED** [review]

**Pull request merged**